Code/Framework/AzToolsFramework/AzToolsFramework/Input/QtEventToAzInputMapper.h

```diff
@@ -69,6 +71,12 @@ namespace AzToolsFramework
         //! \param event The underlying Qt event that triggered this change, if applicable.
         void InputChannelUpdated(const AzFramework::InputChannel* channel, QEvent* event);
 
+    protected:
+        //! \ref AzFramework::InputChannelNotifications::GetPriority
+        AZ::s32 GetPriority() const override;
+        //! \ref AzFramework::InputChannelNotifications::OnInputChannelEvent
+        void OnInputChannelEvent(const AzFramework::InputChannel& inputChannel, bool& o_hasBeenConsumed) override;
```

Comment:
update: Here I believe the preferred style is to just name the bus these overrides come from like so...

```
//  AzFramework::InputChannelNotificationBus overrides ...
AZ::s32 GetPriority() const override;
void OnInputChannelEvent(const AzFramework::InputChannel& inputChannel, bool& o_hasBeenConsumed) override;
```

We want the name to match the bus type that is inherited from, these makes searching for the definitions much easier in the file